src/npl/config.yml

```diff
@@ -66,3 +66,8 @@ glofas:
     Simle_v3:
       lat: 27.05
       lon: 87.15
+    # DHM point added later
```

Comment:
would add here a comment why the coordinates are not 0.05 rounded and when using the use_incorrect_coords add a comment that the coords are not rounded and thus using this. As you wouldn't expect this to be the standard